systemtest/src/main/java/io/strimzi/systemtest/resources/KubernetesResource.java

```diff
@@ -307,6 +312,84 @@ public static Service deployBridgeNodePortService(String bridgeExternalService,
             .endSpec().build();
     }
 
+    public static <T extends HasMetadata> NetworkPolicy applyNetworkPolicySettingsForResource(T resource, String name) {
```

Comment:
You don't need the generics, just declare the parameter as `HasMetadata resource`. It would also be good to add javadoc. It wasn't immediately clear to me that the `resource` is something like KafkaConnector. And `name` probably isn't the best name for the 2nd parameter (the name of _what_?)

Finally I think the method name would be clearer with 'allow' it in, since this policy is allowing access